client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -123,24 +230,21 @@ define([
                     };
                 }
                 // return the post saved
-                return operation().then(function (post) {
-                    $rootScope.$broadcast('lb.posts.updated');
-                    // post here doesn't contain the items...
-                    $rootScope.$broadcast('lb.posts.saved', post);
-                    return post;
-                });
+                return operation();
             });
         }
 
         function removePost(post) {
-            return api.posts.remove(post).then(function() {
-                $rootScope.$broadcast('lb.posts.updated');
-                $rootScope.$broadcast('lb.posts.removed', post);
-            });
+            return api.posts.remove(post);
         }
 
         return {
+            posts: posts,
```

Comment:
`posts` and `getPosts` are confusing, do we need both of them ? Can't we rename one ?